src/app/personside/dialogpanel/sendMelding/TekstFelt.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { NyMeldingValidator } from './validatorer';
+import { SkjemaGruppe, Textarea } from 'nav-frontend-skjema';
+import { FormState } from './SendNyMelding';
+import styled from 'styled-components';
+import theme from '../../../../styles/personOversiktTheme';
+
+const StyledSkjemagruppe = styled(SkjemaGruppe)`
+    margin-top: 1rem;
+    textarea {
+        min-height: 9rem;
+    }
+    label {
+        ${theme.visuallyHidden}
+    }
+`;
+
+interface Props {
+    formState: FormState;
+    navn: string;
+    tekstMaksLengde: number;
+    updateTekst: (tekst: string) => void;
+}
+
+function TekstFelt(props: Props) {
+    return (
+        <StyledSkjemagruppe
+            feil={
+                !NyMeldingValidator.tekst(props.formState) && props.formState.visFeilmeldinger
```

Comment:
Håndteres `touched`-logik etc av `visFeilmeldinger` ? Eller kjører vi vårt eget løp mtp skjemavalidering?

Reply:
nope, kjører validering ved send. Er det som var instruksene her forrige gang jeg lagde skjema, men ting endrer seg fort ;-) tenker eventuell omskriving av valideringsregimet eventuellt kan komme i en ny PR..?

Reply:
Er relativt fastsatte regler for validering for ekstern-flater ihvertfall. Men vet ikke hvordan det er med interneflater. Det som er viktig er at modia-plattformen har nogenlunde samme regler tenker jeg

Reply:
Da får du starte med feks fødselsnummervalidering i internflatedekoratoren :-P Men ja, kan sikkert være greit å ta en runde på, men da bør vel hvertfall designeren vår være involvert